From b22c44f4f48d4286f9b9f3c4cdd0f68079ad1d5b Mon Sep 17 00:00:00 2001
From: Jeffy Chen <jeffy.chen@rock-chips.com>
Date: Wed, 22 Aug 2018 21:16:31 +0800
Subject: [PATCH] dt-bindings: rtc: Add rk-timer-rtc

Add documentation for DT binding of Rockchip timer RTC driver.

Change-Id: I066312219d9ed9e7a3e943bf8a60d072d8c9fa36
Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
---
 .../devicetree/bindings/rtc/rtc-rk-timer.txt  | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/rtc/rtc-rk-timer.txt

diff --git a/Documentation/devicetree/bindings/rtc/rtc-rk-timer.txt b/Documentation/devicetree/bindings/rtc/rtc-rk-timer.txt
new file mode 100644
index 000000000000..115124f68a86
--- /dev/null
+++ b/Documentation/devicetree/bindings/rtc/rtc-rk-timer.txt
@@ -0,0 +1,19 @@
+Rockchip rk timer RTC
+
+Required properties:
+- compatible: should be:
+  "rockchip,rk3308-timer-rtc": for Rockchip RK3308
+- reg: base address of the timer register and length of memory mapped region.
+- interrupts: should contain the interrupts for the timer
+- clocks : must contain an entry for each entry in clock-names
+- clock-names : must include the following entries:
+  "timer", "pclk"
+
+Example:
+	rk_timer_rtc: rk-timer-rtc@ff1a0020 {
+		compatible = "rockchip,rk3308-timer-rtc";
+		reg = <0x0 0xff1a0020 0x0 0x20>;
+		interrupts = <GIC_SPI 26 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&cru PCLK_TIMER>, <&cru SCLK_TIMER1>;
+		clock-names = "pclk", "timer";
+	};
-- 
2.35.3

